src/views/Home/index.jsx

```diff
@@ -27,8 +28,13 @@ const Home = () => {
         <Grid item xs={12} md={5}>
           <DiningBalance />
         </Grid>
-        <Grid item xs={12} md={5}>
-          <NewsCard />
+        <Grid container item xs={12} md={5}>
+          <Grid item xs={12}>
+            <LostAndFoundCard />
+          </Grid>
+          <Grid item xs={12}>
+            <NewsCard />
+          </Grid>
```

Comment:
I believe the container was intended to format the Student News card underneath our Lost and Found card like this:
<img width="1728" alt="Screenshot 2024-12-05 at 8 49 14 PM" src="https://github.com/user-attachments/assets/66336e34-196b-4228-adce-bb5e34fcd42a">

Without the container the home page looks like this:
<img width="1728" alt="Screenshot 2024-12-05 at 8 48 48 PM" src="https://github.com/user-attachments/assets/763372a6-c9c1-4be6-b8c5-0eaf30c19679">

Is this the intended functionality?

Reply:
Yes, that is the expected behavior of MUI's `Grid` component. If you want those cards to stack within one Grid item, you can use a `Stack` instead of a nested `Grid`.

Reply:
Fixed in PR: https://github.com/gordon-cs/gordon-360-ui/pull/2476